client/superdesk.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = function() {
     return {
         apps: ['superdesk-planning', 'superdesk.analytics', 'aap.apps'],
-        importApps: ['superdesk-planning', 'superdesk-analytics', '../aap'],
+        importApps: ['superdesk-planning', 'superdesk-analytics', '../index.js','../aap'],
```

Comment:
It might be better to only have `['superdesk-planning', 'superdesk-analytics', '../index.js']`, and in `../index.js` import the `aap` module. This will allow that `index.js` file to control what local source to include.

Reply:
Yea I was wondering about that will give it a go